brave/src/main/java/brave/Tag.java

```diff
@@ -87,38 +87,41 @@ public final void tag(I input, Span span) {
   /**
    * Tags the value parsed from the {@code input}.
    *
+   * @return the tag value if one was parsed.
    * @since 5.11
    */
   // ex void parse(HttpRequest request, TraceContext context, SpanCustomizer span);
-  public final void tag(I input, @Nullable TraceContext context, SpanCustomizer span) {
+  @Nullable public final String tag(I input, @Nullable TraceContext context, SpanCustomizer span) {
```

Comment:
this is compatible but we can't change this again :)

Reply:
It's compatible with source compilation, but I'm pretty sure it's ABI incompatible, meaning if two libraries target different Brave and use this method, one gets `NoSuchMethodError`.

Reply:
Found this tool that can allow the void bytecode to be outputted as well. Seems quite hacky though

https://github.com/infradna/bridge-method-injector